inline_scan.sh

```diff
@@ -115,7 +119,7 @@ main() {
 
 get_and_validate_analyzer_options() {
     #Parse options
-    while getopts ':k:s:r:u:p:a:d:f:i:m:t:PgVh' option; do
+    while getopts ':k:s:u:p:a:d:f:i:m:t:R:PgVh' option; do
```

Comment:
Options `u`, `p`, `t`and `g`do not longer exist. What about removing them from `getopts`?